recipes/Data-Prep-Kit/intro/setup-python-dev-env.md

```diff
@@ -0,0 +1,72 @@
+# Setup a Local Python Dev Environment
+
+## Step-1: Anaconda Python environment
+
+You can install Anaconda by following the [guide here](https://www.anaconda.com/download/).
```

Comment:
We should be telling people to use Ananconda or Miniconda. Please use [Miniforge](https://github.com/conda-forge/miniforge) for the `conda` command.

Reply:
acknowledged